code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -284,7 +284,7 @@
 	return 1
 
 /datum/species/proc/handle_temperature(datum/gas_mixture/breath, var/mob/living/carbon/human/H) // called by human/life, handles temperatures
-	if( (abs(310.15 - breath.temperature) > 50) && !(RESIST_HEAT in H.mutations)) // Hot air hurts :(
+	if(abs(310.15 - breath.temperature) > 50) // Hot air hurts :(
```

Comment:
Yeah, I opted to let breath temperature immunity fall purely to `NO_BREATHE` it also meade 0 sense that having resist heat causes you to be immune to both fire *and* cold temperatures you breathed in.